### PR TITLE
Removed the non-standard HttpStatusCode enumeration from the code base.

### PR DESCRIPTION
**Pull request by wdschei**
It was replaced with the standard `javax.servlet.http.HttpServletResponse` defined constants

### CODE REVIEW

File: repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy [hunk 1]
```diff
@@ -155,16 +145,16 @@ class BasicAuthDelegatingTest extends ReposeValveTest {
...
-        9501      | SC_NOT_IMPLEMENTED          | "status_code=500`component=Rackspace Identity Basic Auth`message=Failed with internal server error"
-        9502      | SC_BAD_GATEWAY              | "status_code=500`component=Rackspace Identity Basic Auth`message=Failed with internal server error"
-        9503      | SC_SERVICE_UNAVAILABLE      | "status_code=500`component=Rackspace Identity Basic Auth`message=Failed with internal server error"
-        9504      | SC_GATEWAY_TIMEOUT          | "status_code=500`component=Rackspace Identity Basic Auth`message=Failed with internal server error"
-        9505      | SC_REQUEST_ENTITY_TOO_LARGE | "status_code=503`component=Rackspace Identity Basic Auth`message=Rate limited by identity service"
-        9506      | SC_TOO_MANY_REQUESTS        | "status_code=503`component=Rackspace Identity Basic Auth`message=Rate limited by identity service"
+        9400      | HttpServletResponse.SC_BAD_REQUEST              | "status_code=500`component=Rackspace Identity Basic Auth`message=Failed with internal server error"
```
Comment (adrianjgeorge) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy]: why did you get rid of the static import?

**Comment by adrianjgeorge** [comment]
not mergable

File: repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy (hunk 1 above)
Comment (wdschei) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy]: It made it more explicit in the table that `SC_TOO_MANY_REQUESTS` is not a member of `HttpServletResponse`, but rather a member of `FilterDirector`.

If this distinction is not necessary, then I can revert this back.

**Comment by wdschei** [comment]
Rebased against master and it should be mergeable again after the tests finish.

File: repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy (hunk 1 above)
Comment (adrianjgeorge) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy]: I'd just leave it statically imported, the source isn't that important, so much as the readability

Comment (wdschei) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/identitybasicauth/BasicAuthDelegatingTest.groovy]: Reverted back to static imports for readability.

**Comment by wdschei** [comment]
Retest this please.

**Comment by wdschei** [comment]
Retest this please.

**Pull request merged**